examples/meta.yaml

```diff
@@ -1,6 +1,5 @@
 categories:
   - messaging
-  - micro
```

Comment:
micro doesn't exist so services ended up at the end after operations which had no entry, which also explains operations and services changes